docs/content/usage/developer-workflow.md

```diff
@@ -192,27 +197,36 @@ You should see no errors.
 ### Apply: deploy your functions to Fission
 
 You can simply use apply to deploy the environment, functions and HTTP
-triggers to the cluster.
+triggers to the cluster. This command will wait for builds of both functions to complete before exiting:
 
 ```bash
 $ fission spec apply --wait
+1 environment created: python
+2 packages created: python-1543660299-o4e9, python-1543660287-byam
+2 functions created: calc-eval, calc-form
+2 HTTPTriggers created: bac55924-03a8-42e1-81b9-8079a8885f3a, f16c8459-3c23-46ad-901f-9312f38cec2a
+--- Build SUCCEEDED ---
+--- Build SUCCEEDED ---
 ```
 
 (This uses your kubeconfig to connect to Fission, just like kubectl.
 See Usage Reference below for options.)
 
 ### Test a function
 
-Make sure your function is working:
+You can check the function is working with `fission fn test` but since this function returns a HTML, it is best to open in browser.
 
 ```bash
 $ fission function test --name calc-form
 ```
 
-You should see the output of the calc-form function.
+Open the URL of the Fission router service suffixed by the name of route at which form function is exposed.
+
+```
+http://$ROUTER_IP/form
```

Comment:
Use `$FISSION_ROUTER` here and guide user to doc: https://docs.fission.io/0.12.0/installation/env_vars/